snap/snapcraft.yaml

```diff
@@ -20,13 +20,13 @@ adopt-info: certbot
 
 apps:
   certbot:
-    command: certbot.wrapper
+    command: bin/python3 $SNAP/bin/certbot
```

Comment:
Unfortunately, Debian 9 doesn't like the `$`:

    root@greek-totem:~# snap install --dangerous certbot_1.8.0-33-gf659124ed_amd64.snap
    error: cannot read snap file: app description field 'command' contains illegal "bin/python3 $SNAP/bin/certbot" (legal: '^[A-Za-z0-9/. _#:-]*$')

It looks like it only became an option in `2.29`: https://github.com/snapcore/snapd/pull/3995.

Is there any hope at all of using a relative path here?

Reply:
Oh. No without an up-to-date snapd it was not working... Given the limitations of what can be used in the command, passing a python command like `bin/python -c "from certbot.main import main; main()"`.

The only remaining option I can think of now is a package call like `bin/python - m certbot`, but I think `__main__.py` is needed in this case.

Reply:
I did that. It works locally and on a Debian 10 DigitalOcean droplet. I could not test the Debian 9 one, because without `snap install core`, the entire `snap` command is not functional as it is unable to download any base snap.

Reply:
True, I suppose this can only be fixed by documentation for Debian 9.